algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -436,7 +437,8 @@ case class CMSItem[K: Ordering](item: K, override val params: CMSParams[K]) exte
   override def ++(other: CMS[K]): CMS[K] = {
     other match {
       case other: CMSZero[_] => this
-      case other: CMSItem[K] => CMSInstance[K](params) + item + other.item
+      case other: CMSItem[K] => CMSItems(List(item, other.item), 2L, params)
```

Comment:
We should probably just add `.toCMS` here. If `maxExactCount == 0`, that will trigger a conversion to `CMSInstance`.